build.gradle

```diff
@@ -485,6 +485,7 @@ project('spring-integration-jmx') {
 		compile project(":spring-integration-core")
 		testCompile "org.aspectj:aspectjweaver:$aspectjVersion"
 		testCompile "com.hazelcast:hazelcast:$hazelcastVersion"
+		testRuntime 'log4j:log4j:1.2.17' // Needed by hazelcast
```

Comment:
FYI: http://docs.hazelcast.org/docs/latest-development/manual/html/Setting_Up_Clusters/Logging_Configuration.html.

So, I think we still can avoid this dependency.

In the Hazelcast extension I use this:
```
jvmArgs "-Dhazelcast.logging.type=slf4j"
```
In the test task definition.

Maybe that will work here as well for log4j2?